pkg/dashboard/distro/default_distro_info.go

```diff
@@ -0,0 +1,11 @@
+// +build !distro
+
+package distro
+
+// Resource declared the distro brand information
+var Resource = map[string]interface{}{
```

Comment:
BTW why do we use `interface{}` instead of `string` here?

Reply:
This is a mistake, and I forget to fix it in the last update. One more round. Sorry for so many times review work.